packages/features/portfolio/src/hooks/use-vault-holdings-table.tsx

```diff
@@ -252,6 +277,18 @@ export const useVaultHoldingsTable = () => {
         },
       ];
 
+      if (!points) {
+        subRowData.push({
+          label: (
+            <FormattedMessage defaultMessage={'Time to Next Reinvestment'} />
+          ),
+          value:
+            network === 'mainnet'
```

Comment:
Don't use string comparison here, use Network.mainnet instead, move this into the helper function as well

Reply:
What're you wanting to move to a helper? This if and the else if subRowData stuff?